rpc_util.go

```diff
@@ -662,6 +663,40 @@ func setCallInfoCodec(c *callInfo) error {
 	return nil
 }
 
+// ParseDialTarget returns the network and address to pass to dialer
+func ParseDialTarget(target string) (net string, addr string) {
```

Comment:
I would prefer to not export this function. It's a helper function for the default dialer, not a general purpose function for users.

Reply:
understood, will update